lib/pushmi_pullyu.rb

```diff
@@ -1,12 +1,63 @@
 require 'pushmi_pullyu/logging'
 
 require 'pushmi_pullyu/cli'
-require 'pushmi_pullyu/config'
 require 'pushmi_pullyu/preservation_queue'
 require 'pushmi_pullyu/version'
 require 'active_support'
 require 'active_support/core_ext'
 
 # PushmiPullyu main module
 module PushmiPullyu
+  LOGFILE = 'log/pushmi_pullyu.log'.freeze
+  PIDDIR  = 'tmp/pids'.freeze
+  PROCESS_NAME = 'pushmi_pullyu'.freeze
+
+  DEFAULTS = {
+    daemonize: false,
+    debug: false,
+    logfile: LOGFILE,
+    minimum_age: 0,
+    monitor: false,
+    piddir: PIDDIR,
+    process_name: PROCESS_NAME,
+    redis: {
+      host: 'localhost',
+      port: 6379,
+      queue_name: 'dev:pmpy_queue'
```

Comment:
Also, total bikeshed: the queue name is a concept independent of redis (we could replace the queue implementation with postgres or something and keep it), so to me it seems like it should be outside the `redis:` key, like the `minimum_age:` queue default.